controls/raddiagram/howto/custom_connectors.md

```diff
@@ -0,0 +1,127 @@
+---
+title: How to populate RadDiagramShape with custom connectors in MVVM scenario
```

Comment:
Usually, the articles in the HowTo section doesn't need the 'How to' part in their names. You can remove it and the title will become "Populate RadDiagramShape with custom connectors in MVVM scenario". However, even so, the title is quite long. Can we shorten it a bit? For example "Using custom connectors in MVVM". I think this is good because the connectors can be used only into the shape's context so if you remove the 'Populate RadDiagramShape with' there shouldn't be a problem.